experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -64,6 +64,16 @@ using ClimaCore: InputOutput
 include("cli_options.jl")
 (s, parsed_args) = parse_commandline()
 
+# modify parsed args for fast testing from REPL #hide
+# parsed_args["coupled"] =  true #hide
+# parsed_args["moist"]= "equil" #hide
+# parsed_args["vert_diff"] =  true #hide
+# parsed_args["rad"] =  "gray" #hide
+# parsed_args["microphy"] =  "0M" #hide
+# parsed_args["energy_check"] = true
+# parsed_args["precip_model"] =  "0M" #hide
+# parsed_args["mode_name"] =  "slabplanet" #hide
+
```

Comment:
I've just found `isinteractive()`, which we can use here as a conditional for running from the REPL, so we can leave this uncommented.